doc/ca65.sgml

```diff
@@ -473,6 +473,23 @@ from the mentioned web page, for more information, see there.
 </itemize>
 
 
+<sect1>4510 mode<p>
+
+The 4510 is a microcontroller that is the core of the Commodore C65 aka C64DX.
+It contains among other functions a slightly modified 65CE02 CPU, to allow
+address mapping for 20 bits of address space (1 megabyte addressable area).
+As compared to the description of the CPU in the System Specification of the
+Commodore C65 aka C64DX prototypes ca65 uses these changes:
+<itemize>
+<item><tt>LDA (d,SP),Y</tt> may also be written as <tt>LDA (d,S),Y</tt>
```

Comment:
From the discussion before I got the notion that one _had_ to use `(d,S)` although you'd prefer `(d,SP)`...